doc/en/goodpractices.rst

```diff
@@ -219,101 +219,4 @@ against your source code checkout, helping to detect packaging
 glitches.
 
 
-Integrating with setuptools / ``python setup.py test`` / ``pytest-runner``
```

Comment:
I would keep this section, perhaps renaming it to: 
```rst
Integrating with setuptools using ``pytest-runner``
```

But we should still remove the `Manual Integration` section.

Reply:
From @jaraco 

> Yes, I'm in favor of removing the section altogether. I don't see any value in keeping a reference to pytest-runner.

Reply:
Oh OK then, I missed that comment. 👍 

In that case let's remove it altogether.